composer.json

```diff
@@ -22,8 +22,8 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
-        "jms/serializer": "^3.20",
-        "jms/metadata": "^2.5",
+        "jms/serializer": "^3.28",
+        "jms/metadata": "^2.6",
```

Comment:
The bump on the metadata package isn't strictly needed, but the serializer package has this as its minimum so figured I'd change it to match.